code/__HELPERS/unsorted.dm

```diff
@@ -1095,36 +1083,54 @@ proc/get_mob_with_client_list()
 		else
 			return zone
 
+/*
+	get_turf(): Returns the turf that contains the atom.
+	Example: A fork inside a box inside a locker will return the turf the locker is standing on.
+	The weird for loop with an empty statement is apparently the fastest way possible to do this.
+*/
 /proc/get_turf(const/atom/O)
-	if (isnull(O) || isarea(O) || !istype(O))
```

Comment:
I really really REALLY don't like this assert here. Just have this fail silently and gracefully when we pass it a null, we don't need that kind of information in our runtime logs.